homeassistant/components/lutron_caseta/binary_sensor.py

```diff
@@ -65,6 +65,7 @@ def is_on(self):
 
     async def async_added_to_hass(self) -> None:
         """Register callbacks."""
+        await super().async_added_to_hass()
         self._smartbridge.add_occupancy_subscriber(
```

Comment:
We only want the occupancy subscriber here. super() is intentionally not called.